.github/workflows/publish-to-website.yml

```
@@ -58,4 +58,4 @@ jobs:
         run: |
           git config --local user.name "github-actions[bot]"
           git config --local user.email "41898282+github-actions[bot]@users.noreply.github.com"
-          git diff --quiet || (git add . && git commit -m "Update Learning Path articles" && git push)
+          git add . && git commit -m "Update Learning Path articles" && git push
```

```suggestion
          git add .
          git diff --staged --quiet || (git commit -m "Update Learning Path articles" && git push)
```
This should pick up new files whilst also preventing the CI failing if there are no changes to push.